tests/test_cli.py

```diff
@@ -258,11 +262,13 @@ def test_lack_of_calling_json_deserializer_when_no_query_appears_for_copying(sel
         mock_json.assert_not_called()
 
     @mock.patch('geospaas_processing.copiers.Copier.delete')
-    @mock.patch('os.path.isfile', side_effect=[True, False, True, False])
-    # The even side effects (the 'True' ones) are associated to the destination and the odd ones are
-    # associated to the source path. It is because 'os.path.isfile' is used for evaluating both
-    # source paths and destination paths.
-    def test_correct_destination_folder_for_all_files_that_are_copied(self, mock_isfile, mock_del):
+    @mock.patch('os.path.isfile', return_value=True)
+    @mock.patch('geospaas_processing.copiers.exists', side_effect=[True, False, True, False])
```

Comment:
```suggestion
    @mock.patch('geospaas_processing.copiers.os.path.exists', side_effect=[True, False, True, False])
```

Reply:
@akorosov it was discussed here: https://github.com/nansencenter/django-geo-spaas-processing/pull/35#discussion_r562654203

We can't do that!